src/Common/src/Interop/Windows/kernel32/Interop.CreateProcess.cs

```diff
@@ -13,20 +13,20 @@ internal partial class Kernel32
     {
         [DllImport(Libraries.Kernel32, CharSet = CharSet.Unicode, SetLastError = true, BestFitMapping = false, EntryPoint = "CreateProcessW")]
         internal static extern bool CreateProcess(
-            [MarshalAs(UnmanagedType.LPTStr)] string lpApplicationName,
+            string lpApplicationName,
             StringBuilder lpCommandLine,
             ref SECURITY_ATTRIBUTES procSecAttrs,
             ref SECURITY_ATTRIBUTES threadSecAttrs,
             bool bInheritHandles,
             int dwCreationFlags,
             IntPtr lpEnvironment,
-            [MarshalAs(UnmanagedType.LPTStr)] string lpCurrentDirectory,
+            string lpCurrentDirectory,
             STARTUPINFO lpStartupInfo,
             [Out] PROCESS_INFORMATION lpProcessInformation
         );
 
         [StructLayout(LayoutKind.Sequential)]
-        internal class PROCESS_INFORMATION
+        internal sealed class PROCESS_INFORMATION
```

Comment:
should we be putting sealed on all our interop structs and classes? why does it help?

Reply:
I have added the sealed to ensure that nobody is inheriting from `STARTUPINFO`. If somebody was inheriting from it, the change from 
`cb = Marshal.SizeOf(this)` to `cb = Marshal.SizeOf<STARTUPINFO>()` would be wrong.

And `PROCESS_INFORMATION` just went along for a ride.

I do not think that the `sealed` makes much performance difference here.

Ideally, these would be converted to blittable structs; but that it is a larger change than I wanted to do right now.